clippy_utils/src/lib.rs

```diff
@@ -559,7 +561,12 @@ pub fn trait_ref_of_method<'tcx>(cx: &LateContext<'tcx>, hir_id: HirId) -> Optio
 }
 
 /// Checks if the top level expression can be moved into a closure as is.
-pub fn can_move_expr_to_closure_no_visit(cx: &LateContext<'tcx>, expr: &'tcx Expr<'_>, jump_targets: &[HirId]) -> bool {
```

Comment:
Can you document what this function actually checks for? What are the conditions that must be met to return true?

Reply:
Should be enough there now. The comment is now as long as the function.